proto/buf/alpha/webhook/v1alpha1/event.proto

```diff
@@ -0,0 +1,59 @@
+// Copyright 2020-2022 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package buf.alpha.webhook.v1alpha1;
+
+import "buf/alpha/registry/v1alpha1/repository.proto";
+import "buf/alpha/registry/v1alpha1/repository_commit.proto";
+import "buf/alpha/registry/v1alpha1/webhook.proto";
+import "google/protobuf/timestamp.proto";
+
+// EventService is the service which receives webhook events from Buf.
+service EventService {
+  // Event is the rpc which receives webhook events.
+  rpc Event(EventRequest) returns (EventResponse);
```

Comment:
Now that the service is back https://github.com/bufbuild/buf/pull/1249#discussion_r911239446 is still somewhat of an open question to me. We should document how `callback_url` will work, but that probably doesn't go into this service definition but in the registry request where you set it.

Reply:
Sounds good. We'll do it here: https://github.com/bufbuild/buf/pull/1264